reverse_each_word.rb

```diff
@@ -0,0 +1,11 @@
+def reverse_each_word(sentence)
+  word_arr = sentence.split(' ')
+  new_arr = []
+  word_arr.collect do |word|
```

Comment:
Since `collect` can return the values from the block, for example:

```ruby
squares = [1,2.3].collect{|i| i*i} # will return [1,4,9]
```

while `each` returns the original array

```ruby
squares = [1,2.3].each{|i| i*i} # will return [1,2,3]
```

In your example you could ditch the variable `new_array` and just do

```ruby
words = word_arr.collect do |word|
  word.reverse
end

words.join(" ")
```